azure/packages/test/end-to-end-tests/src/test/audience.spec.ts

```diff
@@ -77,18 +79,18 @@ describe("Fluid audience", () => {
         );
 
         /* This is a workaround for a known bug, we should have one member (self) upon container connection */
-        const originalSelf = await waitForMyself(services.audience);
+        const originalSelf = await waitForMyself(services.audience, "test-user-id-1");
         assert.notStrictEqual(originalSelf, undefined, "We should have myself at this point.");
 
-        const client2 = createAzureClient("test-id-2", "test-user-name-2");
+        const client2 = createAzureClient("test-user-id-2", "test-user-name-2");
         const { services: servicesGet } = await client2.getContainer(containerId, schema);
 
+        const partner = await waitForMyself(servicesGet.audience, "test-user-id-2");
```

Comment:
Same comment here, maybe add a comment about why we need to await rather than just get.